indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamIndexTaskRunner.java

```diff
@@ -1008,7 +1014,8 @@ public void onFailure(Throwable t)
             log.error(t, "Error while publishing segments for sequenceNumber[%s]", sequenceMetadata);
             handoffFuture.setException(t);
           }
-        }
+        },
+        sequencePersistExecutor
```

Comment:
I withdraw from analyzing execution flows and thread pool relationships here. @jihoonson could you please vet this?

Reply:
Sure, I will take a look soon.

Reply:
This still needs to be reviewed by maintainers of `SeekableStreamIndexTaskRunner`.